internal/controller/clusterinstance_helper.go

```diff
@@ -39,17 +39,17 @@ type SpecialVars struct {
 	ControlPlaneAgents, WorkerAgents int
 }
 
-// SiteData is a special object that provides an interface to the SiteConfig spec fields for use in rendering templates
-type SiteData struct {
-	Site        v1alpha1.SiteConfigSpec
+// ClusterData is a special object that provides an interface to the SiteConfig spec fields for use in rendering templates
+type ClusterData struct {
+	Site        v1alpha1.ClusterInstanceSpec
```

Comment:
Maybe Site can be Spec here? Again, not sure how much you care.

Reply:
This will be addressed in a separate PR (https://issues.redhat.com/browse/CNF-13563).